tools/obscuroscan_v2/frontend/src/stores/rollupStore.js

```diff
@@ -14,11 +15,11 @@ export const useBlockStore = defineStore({
         async fetchCount() {
             this.loading = true;
             try {
-                let response = await fetch( Config.backendServerAddress+'/items/block/latest/');
+                let response = await fetch( Config.backendServerAddress+'/items/rollup/latest/');
                 let data = await response.json();
-                this.blocks.add(data.item);
+                this.rollups.add(data.item);
 
-                console.log("Fetched "+data.item.Number);
+                console.log("Fetched "+data.item.L1ProofNumber);
```

Comment:
Doesn't make a difference but thought I'd mention in case you've not seen (cos it's sexy). They added string interpolation with backticks a little while back so you can write it like this:
```
console.log(`Fetched ${data.item.L1ProofNumber}`);
```